.github/workflows/build-test-manylinux-x86.jsonnet

```diff
@@ -13,19 +13,20 @@ local wheel_name = 'manylinux-x86-wheel';
 local build_wheels_job = {
   'runs-on': 'ubuntu-latest',
   // pad: What is this sgrep-xxx image?
-  container: 'returntocorp/sgrep-build:ubuntu-16.04',
+  container: 'returntocorp/sgrep-build:ubuntu-18.04',
```

Comment:
Can we use a more official container? I have no idea what is this sgrep-build thing.
(sgrep was the old name of semgrep, but I don't even remember building those containers).

Reply:
Yea probably, I think this is the equivalent of the current ocaml-layer image?